linera-views/tests/views_tests.rs

```diff
@@ -131,8 +131,8 @@ impl StateStorage for LruMemoryStorage {
 
     async fn new() -> Self {
         let store = MemoryStore::new_test_store().await.unwrap();
-        let cache_size = 1000;
-        let store = LruCachingStore::new(store, cache_size);
+        let storage_cache_config = DEFAULT_STORAGE_CACHE_CONFIG;
```

Comment:
There is no reason for this local definition. Just use the constant directly please!

Reply:
Ok.